src/spatialdata/_types.py

```diff
@@ -7,6 +7,8 @@
     from numpy.typing import DTypeLike, NDArray
 
     ArrayLike = NDArray[np.float64]
+    IntArray = NDArray[np.int64]  # or any np.integer
```

Comment:
rename?

Reply:
to ArrayLikeInt? Also, in case of ImportError/TypeError ArrayLikeInt is not defined.

Reply:
check for later PR. Changing this throws a bunch of mypy errors